src/otx/core/model/entity/segmentation.py

```diff
@@ -153,11 +153,12 @@ def _customize_outputs(
         )
 
     def _configure_export_parameters(self) -> None:
+        image_size = getattr(self, "image_size", (512, 512))
```

Comment:
Since this is the entity class, it should be clean from checking attribute existence tricks, so I'd create `image_size` attribute in constructor